DEVELOPMENT.md

```diff
@@ -102,6 +102,40 @@ _Note: This test requires at least 5Gi of memory_
 LOCAL_DEVELOPMENT=true SKIP_INITIALIZE=true ./test/e2e-tests.sh
 ```
 
+### Run a single rekt test
+
+You can run a single rekt test instead of the full test suite with:
+
+```shell
+SYSTEM_NAMESPACE=knative-eventing go test -v -tags=e2e -count=1 -run <test_name> -parallel=12 -timeout=30m ./test/rekt
+```
```

Comment:
Can you add a note here indicating that they have to replace `<test_name>` with the name of the test?

Reply:
done @Cali0707